lib/gems/pending/appliance_console/cli.rb

```diff
@@ -55,6 +59,14 @@ def local_database?
       database? && local?(hostname)
     end
 
+    def db_credentials?
```

Comment:
If that's the only place it's used we can keep it all in the same method.

Having `#replication?` enumerate all the dependencies of that task is fine.